source/_components/life360.markdown

```diff
@@ -0,0 +1,286 @@
+
+## {% linkable_title Examples %}
+
+### {% linkable_title Typical configuration %}
+
+{% raw %}
+```yaml
+device_tracker:
+  - platform: life360
+    username: LIFE360_USERNAME
+    password: LIFE360_PASSWORD
+    add_zones:
+        days: 1
+    # MPH, assuming imperial units.
+    # If using metric (KPH), equivalent would be 29.
+    driving_speed: 18
+    interval_seconds: 10
+    max_gps_accuracy: 200
+    max_update_wait:
+      minutes: 45
+    show_as_state: [driving, moving]
+    # Set comm error thresholds so first is not logged,
+    # second is logged as a WARNING, and third and fourth
+    # are logged as ERRORs.
+    warning_threshold: 2
+    error_threshold: 3
+```
+{% endraw %}
+
+### {% linkable_title Circe, Member and Place Filtering Example %}
+
+{% raw %}
+```yaml
+device_tracker:
+  - platform: life360
+    username: LIFE360_USERNAME
+    password: LIFE360_PASSWORD
+    circles:
+      include: [Our Family, Friends]
+    members:
+      exclude: John Doe
+    places:
+      include:
+        - Home
+        - Work (Dad)
+        - Work (Mom)
+        - School (Jane)
+```
+{% endraw %}
+
+### {% linkable_title Example overdue update automations %}
+
+{% raw %}
+```yaml
+automation:
+  - alias: Life360 Overdue Update
+    trigger:
+      platform: event
+      event_type: life360_update_overdue
+    action:
+      service: notify.email_me
+      data_template:
+        title: Life360 update overdue
+        message: >
+          Update for {{
+            state_attr(trigger.event.data.entity_id, 'friendly_name') or
+            trigger.event.data.entity_id
+          }} is overdue.
+
+  - alias: Life360 Update Restored
+    trigger:
+      platform: event
+      event_type: life360_update_restored
+    action:
+      service: notify.email_me
+      data_template:
+        title: Life360 update restored
+        message: >
+          Update for {{
+            state_attr(trigger.event.data.entity_id, 'friendly_name') or
+            trigger.event.data.entity_id
+          }} restored after {{ trigger.event.data.wait }}.
+```
+{% endraw %}
+
+## {% linkable_title Disclaimer %}
+
+It does not appear that Life360 officially supports its REST API for use with other than its own apps. This integration is based on reverse engineering that has been done by the open source community, and an API token that was somehow discovered by the same community. At any time Life360 could disable that token or otherwise change its REST API such that this custom component would no longer work.
```

Comment:
```suggestion
It does not appear that Life360 officially supports its REST API for use with other than its own apps. This integration is based on reverse engineering that has been done by the open source community, and an API token that was somehow discovered by the same community. At any time Life360 could disable that token or otherwise change its REST API such that this component would no longer work.
```

Reply:
Good catch! But actually, now it's an "integration", right?

Reply:
you are right, but a component or integration are the same. Even though we are pushing a bit to only talk about integrations 😄